crates/nu-command/Cargo.toml

```diff
@@ -88,6 +88,9 @@ toml = "0.7"
 unicode-segmentation = "1.10"
 ureq = { version = "2.7", default-features = false, features = ["charset", "gzip", "json", "native-tls"] }
 url = "2.2"
+# uu_cp = { path = "../../../coreutils/src/uu/cp"}
+# 0.0.21 not on crates.io yet?
```

Comment:
There comments can be removed now right? At least I think 0.0.21 is all good on crates.io now.

Reply:
I did this manually, lol.